packages/demo/src/app/app.component.html

```diff
@@ -1,135 +1,441 @@
 <div class="layout-container header-big">
-  <sp-intranet-header siteTitle="Swiss Post Design System" languages="en" logoUrl="/">
+  <ng-template #versionDropdown ngbDropdown>
+      <label class="visually-hidden" for="versionDropdownButton" id="versionDropdown">Version Selection for older documentation versions</label>
+      <button class="btn btn-link dropdown-button" id="versionDropdownButton" ngbDropdownToggle>
+        v{{ version }}
```

Comment:
Showing the demo version here is confusing because it does not match any of the versions actually installed in the projects. I'd rather show the version of the `@post/design-system-styles` package since, for now, this is the base design system package. What do you think?

Reply:
This would work for now because I wrote the versions.json manually. As soon as we're looking to automate this, we'd have to follow the version of the demo/documentation package.json because deployment is following this. What we could do, is declare these two packages as [fixed](https://github.com/changesets/changesets/blob/main/docs/fixed-packages.md) or [linked](https://github.com/changesets/changesets/blob/main/docs/linked-packages.md).

Reply:
I simplified the displayed versions (just showing major updates). We need to discuss how we automate this exactly as it's not easy, but I think I have a better understanding of how we could do it in the future.